package.json

```diff
@@ -12,6 +15,7 @@
   "devDependencies": {
     "jquery": "*",
     "ethereumjs-accounts": "git+https://github.com/tcoulter/ethereumjs-accounts.git",
+    "truffle": "*",
```

Comment:
Truffle isn't built to be a local dependency stored in `node_modules`. It really should be, but that's a feature for later. We should remove it from `package.json`